backend/api/routers/contests.py

```diff
@@ -1,47 +1,26 @@
 import json
-from datetime import datetime
 from typing import Optional
 
 from bson import ObjectId, json_util
 from fastapi import APIRouter, HTTPException
-from pydantic import BaseModel, AnyHttpUrl
 from starlette.requests import Request
 
 
-router = APIRouter(
-    prefix='/contests',
-    tags=['Contests']
-)
-
+router = APIRouter(prefix="/contests", tags=["Contests"])
 
-class Contest(BaseModel):
-    name: str
-    description: str
-    category: str
-    entryCategories: list[str]
-    published: bool
-    deadline: datetime
-    termsAndConditions: list[AnyHttpUrl]
-    acceptedFileFormats: list[str]
-    background: AnyHttpUrl
 
-
-@router.post('/')
-async def post_contest(
-    data: Contest,
-    request: Request
-):
+@router.post("/")
+async def post_contest(request: Request):
     db = request.app.database
 
-    entry_dict = data.model_dump(mode='json')
+    entry_dict = await request.json()
```

Comment:
What the fuck

Reply:
?

Reply:
So-called unification

Reply:
Your favourite

Reply:
That's total destruction, not unification

Reply:
You've literally removed all validation and good fastapi practices

Reply:
At least all endpoints are equally destructed